### PR TITLE
GLT-354 &GLT-393

### PR DESCRIPTION
**Pull request by callunity**
* Add Box validation (GLT-354)
* Refactor Parsley.js validations to remove deprecated methods and eliminate naming collisions (GLT-393)

### CODE REVIEW

**Comment by callunity** [comment]
I've done the validations for Sequencer Partition Container and renamed the x_validation.js files to x_ajax.js files. I'm going to leave Submission alone right now, as it is pretty complex and from what I recall, the TGAC lab doesn't actually use it. Chris, could you please confirm?

**Comment by frankie13** [comment]
:spaghetti:

**Comment by callunity** [comment]
:no_entry: :spaghetti:

**Comment by frankie13** [comment]
:spaghetti: :exclamation:

**Comment by callunity** [comment]
:no_entry: :spaghetti:

File: miso-web/src/main/webapp/pages/editBox.jsp [hunk 1]
```diff
@@ -160,11 +168,11 @@
     </tr>
     <tr>
       <td class="h">Alias:</td>
-      <td><form:input path="alias" class="validateable"/><span id="aliascounter" class="counter"></span></td>
+      <td><form:input id="alias" path="alias" class="validateable"/><span id="aliascounter" class="counter"></span></td>
     </tr>
     <tr>
       <td class="h">Description:</td>
-      <td><form:input path="description" class="validateable"/></td>
+      <td><form:input id="description" path="description" class="validateable"/></td>
```
Comment (frankie13) [inline miso-web/src/main/webapp/pages/editBox.jsp]: ahem *cough cough* 
https://en.wiktionary.org/wiki/validatable

:neckbeard:

**Comment by frankie13** [comment]
:+1:

**Comment by djcooke** [comment]
:+1:

**Pull request merged**